src/elements/content-sidebar/activity-feed/activity-feed/ActivityFeed.js

```diff
@@ -245,7 +246,12 @@ class ActivityFeed extends React.Component<Props, State> {
 
         return (
             // eslint-disable-next-line
-            <div className="bcs-activity-feed" data-testid="activityfeed" onKeyDown={this.onKeyDown}>
+            <div
+                className="bcs-activity-feed"
+                data-resin-component={ACTIVITY_TARGETS.ACTIVITY_FEED}
```

Comment:
The `data-resin-component` for the VersionsSidebar is `preview`. Should we remain consistent between sibling sidebar components?